pkg/quota_management/quota_management_list_config_test.go

```diff
@@ -0,0 +1,148 @@
+package quota_management
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestNewQuotaManagementListConfig(t *testing.T) {
+	tests := []struct {
+		name string
+		want *QuotaManagementListConfig
+	}{
+		{
+			name: "Should return the QuotaManagementListConfig",
+			want: &QuotaManagementListConfig{
+				QuotaList: RegisteredUsersListConfiguration{
+					Organisations:   nil,
+					ServiceAccounts: nil,
+				},
+				QuotaListConfigFile:        "config/quota-management-list-configuration.yaml",
+				EnableInstanceLimitControl: false,
+			},
+		},
+	}
+	for _, tt := range tests {
+		RegisterTestingT(t)
+		t.Run(tt.name, func(t *testing.T) {
+			got := NewQuotaManagementListConfig()
+			Expect(got).To(Equal(tt.want))
+		})
+	}
+}
+
+func TestQuotaManagementListConfig_ReadFiles(t *testing.T) {
+	type fields struct {
+		QuotaList                  RegisteredUsersListConfiguration
+		QuotaListConfigFile        string
+		EnableInstanceLimitControl bool
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		wantErr bool
+	}{
+		{
+			name: "Should sucesfully read the config files without error",
+			fields: fields{
+				QuotaList: RegisteredUsersListConfiguration{
+					Organisations:   nil,
+					ServiceAccounts: nil,
+				},
+				QuotaListConfigFile:        "config/quota-management-list-configuration.yaml",
+				EnableInstanceLimitControl: false,
+			},
+			wantErr: false,
+		},
+		{
+			name: "Should return nil if the file cannot be found",
+			fields: fields{
+				QuotaList: RegisteredUsersListConfiguration{
+					Organisations:   nil,
+					ServiceAccounts: nil,
+				},
+				QuotaListConfigFile:        "fake-file-path",
+				EnableInstanceLimitControl: false,
+			},
+		},
+	}
+	for _, tt := range tests {
+		RegisterTestingT(t)
```

Comment:
please call `RegisterTestingT(t)` before the for loop. no need to call it for all test cases